scripts/instance/mask_rcnn/train_mask_rcnn.py

```diff
@@ -587,14 +589,18 @@ def train(net, train_data, val_data, eval_metric, batch_size, ctx, logger, args)
             if (not args.smdataparallel or dist.rank() == 0) and args.log_interval \
                     and not (i + 1) % args.log_interval:
                 msg = ','.join(['{}={:.3f}'.format(*metric.get()) for metric in metrics + metrics2])
+                batch_speed = args.log_interval * args.batch_size / (time.time() - btic)
+                speed.append(batch_speed)
                 logger.info('[Epoch {}][Batch {}], Speed: {:.3f} samples/sec, {}'.format(
-                    epoch, i, args.log_interval * args.batch_size / (time.time() - btic), msg))
+                    epoch, i, batch_speed, msg))
                 btic = time.time()
+        if speed:
+            avg_batch_speed = sum(speed)/len(speed)
```

Comment:
nit: can use np.average to compute

Reply:
I'll leave it as is; users can switch to np if need be :)